samples/bluetooth/mesh/light/sample.yaml

```diff
@@ -14,3 +14,4 @@ tests:
       - nrf51dk_nrf51422
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuappns
+      - thingy53_nrf5340_cpuapp
```

Comment:
This should be added to `platform_allow` too

Reply:
That's right i missed that.